### PR TITLE
C sharp oop 1

### PR DESCRIPTION
**Pull request by Moment-schastia**

### CODE REVIEW

File: Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        //float foodPortion = float.Parse(Console.ReadLine());
+        //Console.Write("Enter a quantity of drink bottles - ");
+        //float drinkBottle = float.Parse(Console.ReadLine());
+        //Console.Write("Enter a quantity of souces - ");
+        //float souce = float.Parse(Console.ReadLine());
+
+
+        //AnOrder(foodPortion, 98.4f, drinkBottle, 28.7f, souce, 0.8f, 12);
+
+        //void AnOrder(float foodPortion, float foodCost, float drinkBottle, float drinkCost, float souce, float souceCost, float discount)
```
Comment (dnnyhlwdplc) [inline Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs]: Method could be simplified

File: Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        //{
+
+        //    float theSumCost = (foodPortion * foodCost) + (drinkBottle * drinkCost) + (souce * souceCost);
+        //    float theDiscount = theSumCost * discount / 100;
+
+        //    Console.WriteLine($"The total cost - {theSumCost}. The discount - {theDiscount}. The final amount to the payment - {theSumCost - theDiscount}");
```
Comment (dnnyhlwdplc) [inline Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs]: As this method is general it should return only result of operation

File: Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        //Console.Write("Enter a quantity of souces - ");
+        //string souce = Console.ReadLine();
+
+
+        //AnOrder(foodPortion, "98.4", drinkBottle, "28.7", souce, "0.8", "12", foodName, drinkName, souceName);
+
+        //void AnOrder(string foodPortion, string foodCost, string drinkBottle, string drinkCost, string souce, string souceCost, string discount, string foodName, string drinkName, string souceName)
+        //{
+
+        //    float theSumCost = (float.Parse(foodPortion) * float.Parse(foodCost)) + (float.Parse(drinkBottle) * float.Parse(drinkCost)) + (float.Parse(souce) * float.Parse(souceCost));
```
Comment (dnnyhlwdplc) [inline Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs]: reuse created method in task 1

File: Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+        //    float theDiscount = theSumCost * float.Parse(discount) / 100;
+
+        //    Console.WriteLine($"The cost of one {foodName} - {foodCost}");
```
Comment (dnnyhlwdplc) [inline Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs]: The cost of {name} number of items {items} is {cost}

File: Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+        //Console.Write("Enter the wanted souce - ");
+        //string souceName = Console.ReadLine();
+
+
+        //AnOrder("98.4", "28.7", "0.8", "12", foodName, drinkName, souceName);
+
+        //void AnOrder(string foodCost, string drinkCost, string souceCost, string discount, string foodName, string drinkName, string souceName, string foodPortion = "12", string drinkBottle = "14", string souce = "8")
+        //{
+
+        //    float theSumCost = (float.Parse(foodPortion) * float.Parse(foodCost)) + (float.Parse(drinkBottle) * float.Parse(drinkCost)) + (float.Parse(souce) * float.Parse(souceCost));
```
Comment (dnnyhlwdplc) [inline Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs]: reuse 1st method

File: Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+        //    Console.WriteLine($"The cost of one {drinkName} - {drinkCost}");
+        //    Console.WriteLine($"The cost of one {souceName} - {souceCost}");
+        //    Console.WriteLine($"The total cost - {theSumCost}. The discount - {theDiscount}. The final amount to the payment - {theSumCost - theDiscount}");
+        //    Console.ReadLine();
+        //}
+    }
+    public void Task_4()
+    {
+        //Console.Write("Enter length of the side: ");
+        //int sideLength = int.Parse(Console.ReadLine());
```
Comment (dnnyhlwdplc) [inline Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs]: add more params

File: Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+        //Console.ReadLine();
+
+        //PentagonPerimeterCaunter(sideLength, sideLength, sideLength, sideLength, sideLength);
+        //SquarePerimeterCaunter(sideLength, sideLength, sideLength, sideLength);
+        //TrianglePerimeterCaunter(sideLength, sideLength, sideLength);
+
+        //void PentagonPerimeterCaunter(params int[] sidesInArray)
```
Comment (dnnyhlwdplc) [inline Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs]: rename to PerimeterCount

Comment (dnnyhlwdplc) [reply Nikita_Konkov_Application/CSharpOOP/CSharpOOP/Program.cs]: Return double

**Comment by Moment-schastia** [comment]
did not find comments 
Screenshot_1
Screenshot_2

**Review by dnnyhlwdplc — APPROVED** [review]

**Pull request merged**